modules/code-editor/src/views/full/store/api.ts

```diff
@@ -69,7 +69,7 @@ export default class CodeEditorApi {
       const { data } = await this.axios.post('/mod/code-editor/readFile', file)
       return data.fileContent
     } catch (err) {
-      this.handleApiError(err, 'Could not check if file already exists')
+      this.handleApiError(err, 'Error while reading file')
```

Comment:
Maybe it would be best if `handleApiError` would re-throw the error? That way, we don't have to handle undefined behavior everywhere we make an API call?

Reply:
That handler handles toast errors for all endpoints, if we re-throw, then we'd have to add more handlers for user experience. This situation doesn't happen often enough to change more code there